apps/meeting/src/components/DeviceSelection/CameraDevices/index.tsx

```diff
@@ -24,6 +25,9 @@ const CameraDevices = () => {
       <StyledInputGroup>
         <QualitySelection />
       </StyledInputGroup>
+      <StyledInputGroup>
+        <VideoFilterSelection />
```

Comment:
We have to conditionally render this if background blur is checked else the `BackgroundBlurProvider` will not be in the tree and this will error out.